test/remote/gateways/remote_adyen_test.rb

```diff
@@ -1379,6 +1379,46 @@ def test_successful_authorize_with_sub_merchant_sub_seller_data
     assert_success response
   end
 
+  def test_successful_authorize_with_lvl_2_data
```

Comment:
💬 
I prefer to use the word level instead 'lvl', and It's just matter of personal taste, looks better use words instead numbers in functions